packages/state/src/migrations.test.ts

```diff
@@ -160,4 +160,22 @@ describe("migrations", () => {
       });
     });
   });
+
+  test("8", () => {
+    expect(
+      accountsMigrations[8]({
+        items: [
+          { ...mockMnemonicAccount(0), isVerified: false },
```

Comment:
please update the mockMnemonicAccount to accept Partial<MnemonicAccount> as the second argument instead

Reply:
this change requires updating mockMnemonicAccount across all codebase, so let it be like this for now